bin/cron/process_forms

```diff
@@ -68,6 +68,9 @@ def process_batch_of_forms
         notified_at: DATE_FOR_ERRORS
       )
       next
+    rescue Exception => e
+      ChatClient.log "Unable to process form #{form[:id]} because #{e.message}."
+      raise e
```

Comment:
I think we want a honeybadger error in the case of a parse error, since it's not an expected normal code path. We'd want to make sure we see the parse error and fix it so the form is processed. The `AbortFormError` is intended to drop a form that we know we can no longer process. Currently it's only used for [prize claims when the prize is out](https://github.com/code-dot-org/code-dot-org/blob/66d3bb58af12e0c89a95329a7c63a7141c243c43/pegasus/helpers/prize_helpers.rb#L21), which we haven't even used since HOC 2015.